Makefile

```diff
@@ -6,7 +6,7 @@ TEST_LOCATION	= ./...
 TEST_CMD		= go test -v -mod=vendor -cover ${TEST_LOCATION}
 TEST_CMD_COV	= ${TEST_CMD} -covermode=count -coverprofile=coverage.out
 
-VERSION			= v1.0.0
+VERSION         = `git describe --tags $(git rev-list --tags --max-count=1)`
```

Comment:
Equal sign should be aligned on the same column as the others.

Reply:
fixed